File: app/src/main/java/com/mnj/java_solutions/ElementMOreThan25.java
package com.mnj.java_solutions;

public class ElementMOreThan25 {

    void findSpecialInteger() {
        int[] a = {1, 2, 2, 6, 6, 6, 6, 7};

        int quarter = a.length / 4;

        for (int i = 0; i < a.length - quarter; i++) {
            if (a[i] == a[i + quarter])
                System.out.println("==>> Element " + a[i] + " appears more than %25 of time");
        }

    }
}

File: app/src/main/java/com/mnj/java_solutions/ElementMoreThan25.kt
package com.mnj.java_solutions

class ElementMoreThan25 {
    fun findSpecialInteger() {
        val a = intArrayOf(1, 2, 2, 6, 6, 6, 6, 7)
        val quarter = a.size / 4
        for (i in 0 until a.size - quarter) {
            if (a[i] == a[i + quarter]) println("==>> Element " + a[i] + " appears more than %25 of time")
        }
    }
}
